DOCS-6712-fixed doc bugs (#17)
mulesoft/docs-service-mesh: modules/ROOT/pages/manage-apis-with-service-mesh.adoc

image::manage-service-mesh-breadcrumb.png[]

After you configure Anypoint Service Mesh, you can manage both Mule and non-Mule applications from Anypoint API Manager, including applying policies to secure your microservices. 

> Not all policies available in Anypoint Platform are applicable to Anypoint Service Mesh. Only a subset of those policies apply. Additionally, the configurations within each of the policies that are applicable to Anypoint Service Mesh also differ as compared with the policies that are applicable to Anypoint Platform.

Reviewer: ```suggestion
Not all policies available in Anypoint Platform are applicable to Anypoint Service Mesh. Only a subset of those policies apply. Additionally, the configurations within each of the policies that are applicable to Anypoint Service Mesh also differ from the policies that are applicable to Anypoint Platform.
```

Author: Done! Thanks!
